//package com.sincetimes.redwar.game.comnunicate.protocol;

import "MsgTransaction.proto";

message RceUpdateMissions
{
	optional string planetId = 1;
	optional string sku = 2;
	optional string acton = 3;//newState
	optional int32 state = 4;
	optional MsgTransaction transaction = 5;
}

